cmd/nvidia-device-plugin/main.go

```diff
@@ -100,6 +100,11 @@ func main() {
 			Usage:   "the desired strategy for passing device IDs to the underlying runtime:\n\t\t[uuid | index]",
 			EnvVars: []string{"DEVICE_ID_STRATEGY"},
 		},
+		&cli.BoolFlag{
+			Name:    "gdrcopy-enabled",
+			Usage:   "ensure that containers are started with NVIDIA_GDRCOPY=enabled",
```

Comment:
Question -- since the `NVIDIA_GDRCOPY=enabled` envvar only gets set when a non-CDI device list strategy is configured, would it make sense to omit this implementation detail from the usage string? What about a more generic usage string, like "ensure that containers get access to the GDRCopy device nodes". This comment also applies to the `gds-enabled` and `mofed-enabled` flags.